core/frontend/src/App.vue

```diff
@@ -59,6 +59,12 @@
             </v-list-item-icon>
             <v-list-item-title>Main</v-list-item-title>
           </v-list-item>
+          <v-list-item to="/logs">
```

Comment:
We should break this here with a line to divide in a "Vehicle" category and use a different icon other than home.
Maybe:
![image](https://user-images.githubusercontent.com/1215497/131605879-ec9d4ae6-57bd-46d8-afcf-3b8f5365849a.png)

Reply:
Didn't found this on Material Design Icons.

Think I will use the `file-multiple`:

![image](https://user-images.githubusercontent.com/6551040/131674237-a59cf136-a6bf-4b8b-9d77-ad7e909748ab.png)